rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -321,6 +321,48 @@ public LockWhitelist getLockWhitelist() {
         return lockWhitelist;
     }
 
+    public Coin getFeePerKb() {
+        if (feePerKb != null) {
+            return feePerKb;
+        }
+
+        feePerKb = safeGetFromRepository(FEE_PER_KB_KEY, data -> BridgeSerializationUtils.deserializeCoin(data, STARTING_FEE_PER_KB));
```

Comment:
Default value should be get at the `BridgeSupport` level. This should just be dummy storage.

Reply:
This also saves the default value to the blockchain even if it hasn't been changed. We're usually keeping storage empty unless there's a change.